cronjobs/calculate_monthly_activity.py

```
@@ -0,0 +1,69 @@
+        id_of_month = 12
+        year -= 1
+    else:
+        id_of_month -= 1
+    check_and_save(id_of_month, year)
+
+
+def month_was_saved(activity):
+    if len(activity) > 0:
+        return True
+
+    return False
```

Ruff

Return the condition `len(activity) > 0` directly

Replace with `return len(activity) > 0`

(SIM103)